welder-web.spec

```diff
@@ -0,0 +1,25 @@
+Name: welder-web
+Version: 0
+Release: 0
```

Comment:
> You want it to set the version based on the version in the package.json correct

Yes. Is it possible to just execute package.json with nodejs and pass something like --version?